homework/v.paslavskiy/Hometask1.1.py

```diff
@@ -0,0 +1,101 @@
+name = None
+answer1 = None
+answer2 = None
+answer3 = None
+answer4 = None
+answer5 = None
```

Comment:
would be more convenient to use meaningful names for variables. like
answer_age
answer_name
etc.

Also there's no need to define variables upfront. It's not necessary in python.